### ISSUE TITLE
Release jobs are failing

### ISSUE THREAD

**Issue by tmatsuo** [googleapis/releasetool#467]
I'm not 100% sure, but I found some suspicious logs:

```
Processing chore(main): release Google.Api.Gax 4.3.0-beta01: [https://github.com/googleapis/gax-dotnet/pull/664](https://www.google.com/url?q=https://github.com/googleapis/gax-dotnet/pull/664&sa=D)
> Creating the release.
ValueError('No releases found within pull request')
```

**Comment by amanda-tarafa** [comment]
FYI @jskeet who can give better details, maybe wait until he confirms before closing, just in case.

But I think this is fine. We had a typo in one of our releases and then had to trigger some things by hand, that may have confused release-please.
The package has been released properly: https://www.nuget.org/packages/Google.Api.Gax/4.3.0-beta01

**Comment by jskeet** [comment]
Yes, it's fine to close this.

**Comment by jskeet** [comment]
For some reason the PR still had "pending" as well as "published" labels. I've removed "pending". Everything after line 3 looks like it's about Ruby though.

**Comment by chingor13** [comment]
Fixed in #472
